x-pack/legacy/plugins/reporting/export_types/printable_pdf/types.d.ts

```diff
@@ -6,9 +6,9 @@
 
 import { Request } from 'hapi';
 import { LayoutInstance } from '../common/layouts/layout';
-import { ConditionalHeaders, KbnServer } from '../../types';
+import { ConditionalHeaders, KbnServer, JobDocPayload } from '../../types';
 
-// NOTE: this does not extend the main Params
```

Comment:
The main `JobParams` type should go away. Nothing (correctly) uses it or extends it as the export types have each gotten their own definitions.